LICENSES/Apache-2.0.txt

```diff
@@ -1,73 +0,0 @@
-Apache License
```

Comment:
We actually need both. I checked the [reuse license issue in the Istio repo](https://github.com/kyma-project/istio/issues/307). In the [linked docs](https://sap.github.io/fosstars-rating-core/oss_rules_of_play_rating.html) there is a section **Why do I need to specify a license in the root folder of my project and as well in the LICENSES folder?**.
There it is explained that we need it on both levels.
I think adapting repos for reuse is another task, that we will do for istio module and api-gateway. I'D not consider it part of this task, since there are more steps to do.